builder/nodejs.yaml

```diff
@@ -0,0 +1,8 @@
+steps:
+  - name: 'gcr.io/gcp-runtimes/nodejs-gen-dockerfile:latest'
+    args: ['--base-image', 'gcr.io/google_appengine/nodejs:latest']
```

Comment:
I've updated it to use `google-appengine`.  I've also updated `gcr.io/google_appengine/debian8` to `gcr.io/google-appengine/debian8`.